src/server/modules/sayHello/user/resolvers.ts

```diff
@@ -0,0 +1,5 @@
+export const resolvers: any = {
```

Comment:
You should follow the structure: `/src/server/modules/<module_name>/` where <module_name> is, for example, "user", "company", etc....